connection.go

```diff
@@ -468,15 +471,17 @@ func (conn *Connection) dial() (err error) {
 	return
 }
 
-func pack(h *smallWBuf, enc *msgpack.Encoder, reqid uint32, req Request, res SchemaResolver) (err error) {
+func pack(h *smallWBuf, enc *msgpack.Encoder, reqid uint32,
+	req Request, streamId uint32, res SchemaResolver) (err error) {
 	hl := h.Len()
 	h.Write([]byte{
 		0xce, 0, 0, 0, 0, // Length.
-		0x82,                      // 2 element map.
+		0x83,                      // 3 element map.
 		KeyCode, byte(req.Code()), // Request code.
 		KeySync, 0xce,
 		byte(reqid >> 24), byte(reqid >> 16),
 		byte(reqid >> 8), byte(reqid),
+		KeyStreamId, byte(streamId),
```

Comment:
It looks like we should find a way do not do it if unnecessary.

Reply:
Done:

https://github.com/tarantool/go-tarantool/blob/a027a592265114bd5a0e563c30a418d8671d809a/connection.go#L478-L494